app/navigation/Navigator.scala

```diff
@@ -63,6 +63,11 @@ class Navigator @Inject()() {
     case _ => _ => routes.IndexController.onPageLoad()
   }
 
+  private def getEntryDetails(answers: UserAnswers): Call = answers.get(CustomsRegulationTypePage) match {
```

Comment:
I think this is the wrong way round. Want to have a chat about it?

Reply:
This review comment is incorporated.